css/main.css

```diff
@@ -325,3 +325,78 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.footer p {
+  font-family: "Inter", Arial, Helvetica, sans-serif;
+  font-weight: 700;
+  font-style: normal;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  margin-top: 120px;
+  margin-inline: 24px;
+  color: #172b4d;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 40px;
+  margin-top: 40px;
+  color: #979493;
+}
+
+.form input {
+  font-family: "Inter", Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+
+  /* width: 297px; */
+  width: 327px;
```

Comment:
- [ ] It is one of the best practices to remove all commented code to make the work clean.